package.json

```
@@ -106,7 +106,7 @@
     "reflect-metadata": "^0.1.13",
     "rxjs": "^7.3.0",
     "swagger-ui-express": "^4.1.4",
-    "typechain": "^5.0.0",
+    "typechain": "^7.0.1",
```

```suggestion
    "typechain": "5.2.0",
```
https://github.com/energywebfoundation/iam-client-lib/blob/45db11a8679d1d0af31a5d5a9e84363fcebf1a1d/package.json#L142